common/src/test/kotlin/web5/sdk/common/ConvertTest.kt

```diff
@@ -17,13 +19,199 @@ class ConvertTest {
     }
 
     @Test
-    fun toZBase32() {
-      // TODO: use test vectors from ZBase32Test
-      val output = Convert("a").toZBase32()
-      assertEquals("cr", output)
+    fun toBase64Url_Base64Kind() {
+      val input = "foo"
 
-      val output2 = Convert("cr", EncodingFormat.ZBase32).toStr()
-      assertEquals("a", output2)
+      val output = Convert(input, EncodingFormat.Base64Url).toBase64Url()
+      assertEquals(input, output)
     }
+
+    @Test
+    fun toBase64Url_UnsupportedEncodingFormat() {
+      val input = "foo"
+
+      assertFailsWith<UnsupportedOperationException> {
+        Convert(input, EncodingFormat.Base58Btc).toBase64Url()
+      }
+    }
+
+    @Test
+    fun toBase64Url_UnsupportedValueType() {
+      val input = 1
+
+      assertFailsWith<UnsupportedOperationException> {
+        Convert(input, EncodingFormat.Base58Btc).toBase64Url()
+      }
+    }
+
+    @Test
+    fun toBase58Btc_Base58Kind() {
+      val input = "foo"
```

Comment:
If you're so inclined, here's a way in which we've parameterized tests in the past https://github.com/TBD54566975/web5-kt/blob/48ce6764163b32714be7d8282bfad0678f49debb/dids/src/test/kotlin/web5/sdk/dids/methods/web/DidWebTest.kt#L20